src/handlers/notebook.ts

```diff
@@ -76,7 +77,8 @@ export class NotebookHandler implements IDisposable {
     const codeCell = cell as CodeCell;
     const editorHandler = new EditorHandler({
       debuggerService: this._debuggerService,
-      editor: codeCell.editor
+      editor: codeCell.editor,
+      idCell: cell.model.id
```

Comment:
We should now be able to remove references to the temporary cell ids and instead use the editor finder from https://github.com/jupyterlab/debugger/pull/441?